### DISCUSSION TITLE
Outlook (O365) not remembering default application to open .jpg files

### DISCUSSION THREAD

**Discussion by alex8654** [d2phap/ImageGlass#504] [General] [answered]
Windows 10 1809
Imageglass 6.0.12.29

### To Reproduce
1. Open Outlook (O365 verison is equivalent to Oultlook 2019)
2. open email with .jpg file attached
3. it will ask which application you want to use, then I tick the always use this application for this filetype
4. try to open another .jpg file from the same email, I get asked again for the same thing as above

### Actual behavior:
file associations are not working inside Outlook

### Expected behavior:
File association follows the regular Windows association, which works in Windows Explorer

**Comment by jgodfrey** [comment]
On the surface, this sounds like an Outlook-related issue. What happens if you try to associate a different image viewer (not ImageGlass) to images inside of Outlook? Is that viewer also not remembered? If that's the case, I'm not sure what ImageGlass can do about it.

**Comment by d2phap** [comment]
Hi @alexbleicker 

Probably it's Outlook-related issue in your machine.
However, you can try changing default photo viewer app to MS Photos, then open ImageGlass, go to Settings > File Associations, click on `Set as Default photo viewer` button to register the supported extensions again. After that, change default photo viewer back to ImageGlass.

**Comment by alex8654** [comment, marked as answer]
it was windows related issue in the end, I followed this guide and deleted the registry key associated with .jpg, restart and then did the file association procedure again, it worked fine.

https://www.thewindowsclub.com/outlook-keeps-asking-how-do-you-want-to-open-this-file
